services/capacitivebutton.md

```diff
@@ -0,0 +1,25 @@
+# Capacitive Button
+
+    identifier: 0x1865adc9
+    tags: 8bit
+    group: button
+
+A configuration service for a capacitive push-button.
+
+## Registers
+
+    rw down_threshold: u0.16 / @ inactive_threshold
```

Comment:
so the idea for up/down is that if the current value is between them, then the button retains previous state? once it crosses into say active range, then after a few frames it becomes active, and we wait until it crosses back into inactive for a few frames to de-activates it?

Reply:
right, we only need a active threshold

Reply:
CODAL uses a single threshold

Reply:
https://github.com/lancaster-university/codal-core/blob/master/source/drivers/TouchButton.cpp